### PR TITLE
Bug 1202403 - Use langpack.channel for the Get More Languages link

### PR DESCRIPTION
**Pull request by stasm**
https://bugzilla.mozilla.org/show_bug.cgi?id=1202403

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0057170bbfe23a4ebf383de7feb95c99f28a496c

File: apps/settings/js/panels/languages/languages.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ define(function(require) {
     },
     showMoreLanguages: function() {
       SettingsCache.getSettings(function(result) {
-        var version = result['deviceinfo.os'];
+        var version = result['langpack.channel'];
```
Comment (gasolin) [inline apps/settings/js/panels/languages/languages.js]: where's the value comes from? will it be provided by gecko, or should we preset a value for it?

Comment (stasm) [inline apps/settings/js/panels/languages/languages.js]: The value is controlled by https://github.com/mozilla-b2g/gaia/blob/899860aa2f58cb2597405cd3f8572a938e7ca384/build/config/common-settings.json#L134

Comment (gasolin) [inline apps/settings/js/panels/languages/languages.js]: ok!

**Pull request merged**